cluster/src/main/java/org/apache/iotdb/cluster/server/member/DataGroupMember.java

```diff
@@ -776,14 +774,28 @@ private TSStatus executeNonQueryPlanWithKnownLeader(PhysicalPlan plan) {
       boolean hasCreated = false;
       try {
         if (plan instanceof InsertPlan
-            && status.getCode() == TSStatusCode.TIMESERIES_NOT_EXIST.getStatusCode()
             && ClusterDescriptor.getInstance().getConfig().isEnableAutoCreateSchema()) {
-          hasCreated = createTimeseriesForFailedInsertion(((InsertPlan) plan));
+          if (plan instanceof InsertRowsPlan || plan instanceof InsertMultiTabletPlan) {
```

Comment:
How about put these 10 lines to a new function?

Reply:
Actually, there is only 5 lines to put together as a function, I think it is of no need.

Reply:
It doesn't matter how many lines of code you have. In my opinion, these ten or so lines of code are lower than processPlanLocally, createTimeseries, etc. If you don't think so, then it's okay with me. It's about everyone's code style.